cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -187,6 +207,69 @@ protected Future<Pod> pod(Integer podId) {
     private final ConcurrentHashMap<String, RestartContext> podToContext = new ConcurrentHashMap<>();
     private Function<Pod, RestartReasons> podNeedsRestart;
 
+    private HttpClientOptions getHttpClientOptions() {
+        return new HttpClientOptions()
+                .setLogActivity(true)
+                .setVerifyHost(true)
+                .setSsl(true)
+                .setVerifyHost(true)
+                .setPemTrustOptions(pto)
+                .setKeyCertOptions(pkco);
+    }
+
+    protected <T> Future<Map<String, Object>> doGet(String host) {
+        LOGGER.debugCr(reconciliation, "Making GET request to {}", BROKER_STATE_REST_PATH);
+        return HttpClientUtils.withHttpClient(vertx, getHttpClientOptions(), (httpClient, result) ->
+                httpClient.request(HttpMethod.GET, BROKER_STATE_HTTPS_PORT, host, BROKER_STATE_REST_PATH, request -> {
+                    if (request.succeeded()) {
+                        request.result().setFollowRedirects(true)
```

Comment:
AFAIK the `setFollowRedirects` is true by default?

Reply:
It didn't seem to work if I don't set the setFollowRedirects to true when I was testing. Also from vertx doc, I didn't see it mentioning that it is set to true by default but possible I missed it.